haskell/nixpkgs.bzl

```diff
@@ -297,6 +298,7 @@ haskell_toolchain(
             haddock_flags = repository_ctx.attr.haddock_flags,
             repl_ghci_args = repository_ctx.attr.repl_ghci_args,
             locale_archive = locale_archive,
+            locale = repr(repository_ctx.attr.locale) or [],
```

Comment:
This looks suspicious. Isn't `locale` an `attr.string()`? In fact, if it has a default, why do you need the `or` clause?